tests/test_update.py

```diff
@@ -17,6 +26,95 @@
     clone_package,
 )
 
+@record_requests_module
+@record_git_module
+@record_tempfile_module
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="packit.utils.run_command_remote", decorate=Simple.decorator_plain()
```

Comment:
What about using `Simple.decorator_plain()` by default? Will there be a problem with a different execution time of the `decorator_plain()`?

Reply:
There is no default decorator. I did not thing about that because it may be dangerous.  
 - `ObjectStorage` - the most generic. It uses pickle module to store objects, so everything serializable could be serialized, but it is not readable inside yaml format)
 - `Simple` can be used just for simple python types (dict, list, str, nums) but in case there is some object, it will lead to invalid python objects inside yaml, so that it cannot be loaded then back.
 - `Void` what stores empty value as output, so this is more less just dummy, but the most flexible from perspective of breaking anything :-)

So that's why it is very hard to define what is the best default decorator if not given because each of them have some benefits and cons :-)

Reply:
Can we choose for the user? (Depending on the value we get.)

Reply:
> Can we choose for the user? (Depending on the value we get.)

Hmm, seems like very interesting idea, may create some another class, e.g. GuessOutputType, what will decide the best store option. Question is if I'm able to decide and when is the best time to decide, due to complicated structure, with CaseExecutors and execution time, but I think, it should be possible. We'll see